tv_apps/browser/js/awesomescreen.js

```diff
@@ -572,6 +581,49 @@ var Awesomescreen = {
     evt.target.classList.add('active');
   },
 
+  handleTopListKeydown: function awesomescreen_handleTopListKeydown(ev) {
+    if (ev.keyCode === KeyEvent.DOM_VK_RETURN) {
+      this.clickTopAction(ev);
+    }
+  },
+
+  handleTopListKeyup: function awesomescreen_handleTopListKeyup(ev) {
```

Comment:
Since you've already leveraged `clickTopAction` in `handleTopListKeydown`, why not leverage `clickTopList` in `handleTopListKeyup` as well?